packages/@n8n_io/eslint-config/frontend.js

```diff
@@ -25,5 +25,11 @@ module.exports = {
 		'vue/no-deprecated-slot-scope-attribute': 'error',
 		'vue/no-multiple-template-root': 'error',
 		'vue/v-slot-style': 'error',
+
+		// TODO: remove these
+		'vue/no-unused-components': 'warn',
+		'vue/return-in-computed-property': 'warn',
+		'vue/no-mutating-props': 'warn',
+		'@typescript-eslint/no-floating-promises': 'warn',
```

Comment:
`vue-typed-mixins` is messing up many eslint rules, and removing it made these rules work as expected. 
I'll remove these in [this separate PR](https://github.com/n8n-io/n8n/pull/6242).